### ISSUE TITLE
Generic component not working.

### ISSUE THREAD

**Issue by louiss0** [vuejs/language-tools#2452]
Generic component was not working what did I do wrong

![image](https://user-images.githubusercontent.com/29028601/221323075-d37eac79-de74-408f-a17a-b58a4f7d3a7e.png)

Additional info 

- Volar 1.1.7
- Vue tsc 1.1.7
- I tried takeover mode but that didn't work as well

**Comment by xiaoxiangmoe** [comment]
try generic="T,"

**Comment by louiss0** [comment]
> try `generic="T,"`

I tried it I even upgraded to the latest version of Volar. I think `vue-tsc` is causing the problem. For some reason downloading the latest version messed everything up. Here is the repo https://github.com/louiss0/vue-test-one You can maybe point out a mistake I made.

**Comment by louiss0** [comment]
I have recently learned that generic is experimental. I guess it was used in previous versions. But it's just experimental now. 

![image](https://user-images.githubusercontent.com/29028601/221443015-777ce9c1-a7ac-420f-8cc3-b04698190b15.png)

I turned it on but now I see this.

![image](https://user-images.githubusercontent.com/29028601/221443157-40089143-7678-4ada-8c8b-be80c880ffa8.png)

I think you guys should talk to the [Marko Team](https://twitter.com/MarkoDevTeam) to figure out how to implement this feature.
